contracts/tokenwrapper/NOTICE

```diff
@@ -0,0 +1,13 @@
+Copyright 2022 guorong <dudurong009@gmail.com>
```

Comment:
This needs to be changed to Webb Tools for every similar file please.

Reply:
Removed in https://github.com/webb-tools/protocol-cosmwasm/pull/30/commits/6544b7dbefc1a0673d4588b43569cf5ec73ae239